mathview-lib/build.gradle

```diff
@@ -18,13 +18,12 @@ android {
 
   buildTypes {
     debug {
-      consumerProguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+      //consumerProguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
     }
     release {
       minifyEnabled true
-      shrinkResources true
-      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-      consumerProguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+    //  proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+    //  consumerProguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
why progaurd is removed ?

Reply:
Because of this error 
Error:Default file proguard-android.txt should not be used as a consumer configuration file.
https://stackoverflow.com/questions/26983248/proguard-ignores-config-file-of-library